game/src/test/scala/io/aigar/game/AIStateSpec.scala

```diff
@@ -40,7 +40,7 @@ class AIStateSpec extends FlatSpec with Matchers {
   "WanderingState" should "keep the current cell's target on creation" in {
     val player = new Player(0, Vector2(0f, 0f))
     val cell = player.cells.head
-    cell.target = Vector2(10f, 10f)
+    cell.target = Vector2(100f, 100f)
```

Comment:
Because I've tweak the acceleration and now the target changes after a delta of `1f`.